{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,81 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly. Feel free to remove any
+sections that won't be relevant to reviewers of your PR, including this
+one.
+
+Anything marked with `TODO:` is something that you, the developer, should
+do as part of the PR.
+
+The PR submission process is described in [more detail here.](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html)
+
+### Testing
+- TODO: Include unit, integration, acceptance tests as appropriate
+- TODO: Include accessibility (a11y) tests
+- TODO: Include tests that capture the external-query scaling properties
+- [ ] Check that Database migrations are backwards-compatible
+- [ ] Manually test right-to-left languages and i18n
+  of the changes.
+
+### DevOps assistance
+If it's needed, or even if you suspect it might be needed, do any/all these
+things as early as possible. Don't surprise devops right before merging!
+For example, touching envs/aws.py or or envs/common.py are dead giveaways
+that you'll need assistance when pushing your changes out.
+
+Things that require devops attention include, but are not limited to:
+- configuration changes
+- feature flags
+- migrations
+
+Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
+- TODO: Tag devops on your PR (user `edx/devops`), either for review (more specific to devops = better) or just a general FYI
+- TODO: File a support ticket
+
+FYI: Tag anyone who might be interested in this PR here.
+
+### Documentation considerations
```

Comment:
I'd also expect to see this content in the CONTRIBUTING document rather than the template for the actual PR message.